grep/attributes.yml

```diff
@@ -0,0 +1,2 @@
+grep_path: '/bin/grep'
```

Comment:
There is a risk that this grep is not the grep that we built. `results/last_build.env` will give us the ident for the last built grep executable, and we could generate this attribute.yml from `results/last_build.env` as part of the test steps.